index.js

```diff
@@ -6,6 +6,7 @@ function HashBase (blockSize) {
   Transform.call(this)
 
   this._block = new Buffer(blockSize)
+  this._block.fill(0)
```

Comment:
because it's makes internal state indistinguishable, internal state before and after using should be same